qCC/ccEntityAction.cpp

```diff
@@ -884,7 +889,7 @@ namespace ccEntityAction
 
 			QElapsedTimer eTimer;
 			eTimer.start();
-			pc->applyGaussianFilterToRGB(static_cast<PointCoordinateType>(spatialSigma), static_cast<PointCoordinateType>(sigmaSF), filterParams.applyToSFduringRGB, filterParams.burntOutColorThreshold, parent ? pDlg.data() : nullptr);
+			pc->applyGaussianFilterToRGB(static_cast<PointCoordinateType>(spatialSigma), static_cast<PointCoordinateType>(sigmaSF), filterParams.applyToSFduringRGB,filterParams.filterType & GAUSSIAN_FILTER_TYPES::MEDIAN, filterParams.burntOutColorThreshold, parent ? pDlg.data() : nullptr);
```

Comment:
```suggestion
			pc->applyGaussianFilterToRGB(static_cast<PointCoordinateType>(spatialSigma), static_cast<PointCoordinateType>(sigmaSF), filterParams.applyToSFduringRGB, filterParams.filterType & GAUSSIAN_FILTER_TYPES::MEDIAN, filterParams.burntOutColorThreshold, parent ? pDlg.data() : nullptr);
```

Reply:
And at this point we should probably write only one argument per line...

Reply:
i wanted to use the struct here as well, but i was not sure where to define the struct to be able to do that. i was thinking to define it inside ccPointCloud.h. Maybe later?

Reply:
Why not, as you prefer.

Reply:
but it I'll do it in the next round i think :) i wanna try this in production as well :)